### DISCUSSION TITLE
DCM4CHE for AWS Linux 2

### DISCUSSION THREAD

**Discussion by kevin3791** [dcm4che/dcm4che#1149] [Q&A]
I downloaded DCM4CHE 5.25 and ran it on our AWS Linux 2 ARM server.
I get an error indicating that GLIBC 2.27 cannot be found.

_/mydirectory/lib/linux-aarch64/libopencv_java.so: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by /mydirectory/lib/linux-aarch64/libopencv_java.so)_

Our server only has GLIBC_2.26 available.

Is there a version of DCM4CHE that is designed to work with GLIBC_2.26?

**Comment by gunterze** [comment]
May be related to https://github.com/dcm4che/dcm4che/pull/1125 , so you may try 5.25.1
